recipe/setup_compiler.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [[ ! -d  $SRC_DIR/cf-compilers ]]; then
+    conda create -p $SRC_DIR/cf-compilers -c conda-forge --yes --quiet \
```

Comment:
I'd prefer not to use `--quiet`. It's very unusual to populate conda environments through other means than `meta.yaml`, and if it doesn't even show up in the logs, it's very hard to understand for someone looking at the recipe. Alternatively, we could add a `set -x` at the top.

Reply:
I prefer `--quiet` as opposed very verbose builds. This has been there for a long time and I prefer to keep it that way.

Reply:
> This has been there for a long time

Huh? You're moving build deps from `meta.yaml` into a single function call in the build scripts.

The log prominently showed the build env before anything starts, and now that information would be much much less visible.

Reply:
It was already there in https://github.com/conda-forge/ctng-compilers-feedstock/blob/main/recipe/build.sh#L10

Reply:
I just extended the call to all scenarios